runtime/onert/core/src/ir/Shape.cc

```diff
@@ -100,5 +100,13 @@ Shape permuteShape(const Shape &shape, Layout frontend_layout, Layout backend_la
   return backend_shape;
 }
 
+bool haveUnspecifiedDims(const ir::Shape &shape)
+{
```

Comment:
If rank is 0, should this function return false?

Reply:
thanks! fixed!

Reply:
I mean it would be `return true` if rank is 0

Reply:
It is the question of _"When rank is not specified, can we say dim is also not specified?"_ I think we'd better not to have overlapped area of both test. I will keep it and make better comment on `haveUnspecifiedDims`.

Reply:
If so, the condition seems to be unnecessary. I'm sorry for confusion.

Reply:
haha~ you're correct. :-)